src/components/common/program-section-templates/single-title-description-author-pairs/SingleTitleDescriptionAuthorPairs.module.scss

```
@@ -8,6 +8,9 @@
     width: 100%;
     background-color: c.$white;
 
+    padding: 20px clamp(16px, 3vw, 0px);
```

Stylelint (17.2.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)